lib/modules/user-form-widgets/public/js/main.js

```diff
@@ -0,0 +1,47 @@
+apos.define('user-form-widgets', {
+    extend:    'apostrophe-widgets',
+    construct: function (self, options) {
+        self.play = function ($widget, data, options) {
+            initFilePond();
+            
+            $widget.find('.user-form').each(function () {
+                
+                var $form = $(this);
+                
+                var rules = $form.data('form-validation');
```

Comment:
I think this is the same as: data.formValidation (or data['form-validation'] but maybe that doesn't work...